hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/protocol/DatanodeDetails.java

```diff
@@ -802,7 +802,7 @@ public static final class Port {
      */
     public enum Name {
       STANDALONE, RATIS, REST, REPLICATION, RATIS_ADMIN, RATIS_SERVER,
-      RATIS_DATASTREAM;
+      RATIS_DATASTREAM, HTTP_PORT, HTTPS_PORT;
```

Comment:
Please rename to `HTTP` and `HTTPS`.  None of the other names include "_PORT".

Reply:
Updated .